src/CircleEditor/circle_schema_generated.ts

```diff
@@ -0,0 +1,11908 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * This file is generated with
+ * -
+ * https://github.com/Samsung/ONE/blob/8e41a5031dabc9b6d265cdf1cd8dd8bba598c614/res/CircleSchema/0.4/circle_schema.fbs
```

Comment:
```suggestion
 * - https://github.com/Samsung/ONE/blob/8e41a5031dabc9b6d265cdf1cd8dd8bba598c614/res/CircleSchema/0.4/circle_schema.fbs
```

Reply:
That line is too long, so pr format does not allow this. How shoud I do?

Reply:
Then let's make this as one sentence...

```
/*
 * This file is generated with
 *  https://github.com/Samsung/ONE/blob/8e41a5031dabc9b6d265cdf1cd8dd8bba598c614/res/CircleSchema/0.4/circle_schema.fbs
 *  by flatbuffers v2.0.7
 */
```

Reply:
And do not write this in the copyright statement..!